src/modules/coming-soon/FeaturesList.tsx

```
@@ -0,0 +1,86 @@
+import { Badge } from "@/ui/badge";
+import {
+  Zap,
+  Shield,
+  Globe,
+  Layers,
+  Wallet,
+  Clock,
+} from "lucide-react";
```

ESLint

[error] Replace `⏎··Zap,⏎··Shield,⏎··Globe,⏎··Layers,⏎··Wallet,⏎··Clock,⏎` with `·Zap,·Shield,·Globe,·Layers,·Wallet,·Clock·`

(prettier/prettier)